src/services/stellarExpert.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-underscore-dangle */
+import { NETWORKS } from "config/constants";
+import { logger } from "config/logger";
+import { SearchAssetResponse } from "config/types";
+import { getApiStellarExpertUrl } from "helpers/stellarExpert";
+import { createApiService } from "services/apiFactory";
+
+export const searchAsset = async (asset: string, network: NETWORKS) => {
+  const stellarExpertApi = createApiService({
```

Comment:
will this create a new service instance every time `searchAsset` is called? Should we maybe create a map of static Stellar Expert services per network so we prevent creating multiple service instances? Wdyt?

Reply:
since Stellar Expert has support for only Pubnet and Testnet maybe we don't even need a map, we could probably just create `stellarExpertPubnetApi` and `stellarExpertTestnetApi` on this same file and alternate between them depending if `NETWORKS` is Testnet or not